bindings/python/pinocchio/rpy.py

```diff
@@ -56,4 +56,6 @@ def matrixToRpy(M):
         y = atan2(M[1, 0], M[0, 0])  # alpha
         r = atan2(M[2, 1], M[2, 2])  # gamma
 
-    return np.matrix([r, p, y], np.double).T
+    lst = [[r], [p], [y]]
+    is_matrix = isinstance(M,np.matrix)
```

Comment:
You forgot one space after the comma.

Reply:
Couldn't fix it, but I see @jcarpent handled it. Thanks!